docs/creating-service-bindings/creating-service-binding.md

```diff
@@ -5,7 +5,6 @@ sidebar_position: 1
 # Creating Service Binding
 
 In this section, we consider that the service you want to connect to is already exposing binding metadata by either:
-- Beeing part of the "known" services
+- Being part of the "known" services
 - Exposing binding-metadatas by default
-If it's not the case, please refer to the following section(#)
-
+If it's not the case, please refer to the following section(#)
```

Comment:
I was looking forward to the procedure here. I see a related content for the procedure here: https://github.com/redhat-developer/service-binding-operator/blob/new-userdoc/docs/User_Guide.md#how-it-works
You can use this to write the procedure providing the steps in order.

Question: Is the Draft for the procedure part in progress?